### PR TITLE
gnome-extensions: add page

### PR DESCRIPTION
**Pull request by BhaskarDutta2209**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/gnome-extensions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# gnome-extensions
+
+> Manage gnome extensions from the terminal.
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:3]:
```suggestion
> Manage gnome extensions from the terminal.
> More information: <https://wiki.gnome.org/Projects/GnomeShell/Extensions>.
```

File: pages/linux/gnome-extensions.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Check the version:
+
+`gnome-extensions version`
+
+- List all the installed extensions:
+
+`gnome-extensions list`
+
+- Show info about a particular extension:
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:13]:
```suggestion
- Display information about a specific extension:
```

File: pages/linux/gnome-extensions.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`gnome-extensions info "{{extension name}}"`
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:15]: The help page says `UUID` so:
```suggestion
`gnome-extensions info "{{extension_id}}"`
```

File: pages/linux/gnome-extensions.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Enable a particular extension:
+
+`gnome-extensions enable "{{extension name}}"`
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:19]: We use `_` for variable names:
```suggestion
`gnome-extensions enable "{{extension_id}}"`
```

File: pages/linux/gnome-extensions.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- List all the installed extensions:
+
+`gnome-extensions list`
+
+- Show info about a particular extension:
+
+`gnome-extensions info "{{extension name}}"`
+
+- Enable a particular extension:
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:17]:
```suggestion
- Enable a specific extension:
```

File: pages/linux/gnome-extensions.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+`gnome-extensions enable "{{extension name}}"`
+
+- Disable particular extension:
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:21]:
```suggestion
- Disable specific extension:
```

File: pages/linux/gnome-extensions.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+`gnome-extension disable "{{extension name}}"`
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:23]:
```suggestion
`gnome-extension disable "{{extension_id}}"`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @BhaskarDutta2209, welcome and thanks for your submission. I left some comments.

Please add examples for:
```
gnome-extensions help {{subcommand}}
```
and:
```
gnome-extensions uninstall
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Please @BhaskarDutta2209 revert the changes on files `build.sh` and `test.sh`

**Comment by BhaskarDutta2209** [comment]
I didn't make any changes to the file. I just use chmod to change the permissioning. No code is altered. If needed I can close this PR and create a new one if needed.

File: pages/linux/gnome-extensions.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`gnome-extensions list`
+
+- Display information about a specific extension:
+
+`gnome-extensions info "{{extension_id}}"`
+
+- Display help for any sub-command, for example list:
+
+`gnome-extensions help list`
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:20]:
```suggestion
`gnome-extensions help {{subcommand}}`
```

File: pages/linux/gnome-extensions.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+- List all the installed extensions:
+
+`gnome-extensions list`
+
+- Display information about a specific extension:
+
+`gnome-extensions info "{{extension_id}}"`
+
+- Display help for any sub-command, for example list:
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:18]: Based on the `git` example:
```suggestion
- Display help for a subcommand (like `list`):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/gnome-extensions.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display help for a subcommand (like `list`):
+
+`gnome-extensions help {{subcommand}}`
+
+- Enable a specific extension:
+
+`gnome-extensions enable "{{extension_id}}"`
+
+- Disable specific extension:
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:26]:
```suggestion
- Disable a specific extension:
```

File: pages/linux/gnome-extensions.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+`gnome-extension disable "{{extension_id}}"`
+
+- Uninstall specific extension:
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:30]:
```suggestion
- Uninstall a specific extension:
```

File: pages/linux/gnome-extensions.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+# gnome-extensions
+
+> Manage gnome extensions from the terminal.
+> More information: <https://wiki.gnome.org/Projects/GnomeShell/Extensions>.
+
+- Check the version:
```
Comment (navarroaxel) [inline pages/linux/gnome-extensions.md:6]:
```suggestion
- Display the version:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]
Nice work, thanks @BhaskarDutta2209! 👍🏻

**Comment by BhaskarDutta2209** [comment]
Thanks a lot.

**Pull request merged**